Drupal 10 compatibility

albeortev opened the pull request:
Issue #104 

Updating packages and dependencies to make the project compatible with Drupal 10.

albeortev committed:
Merge branch '2.x' into feature/upgrade-drupal-core-10

albeortev committed:
Issue #104: Upgrade packages for compatibility with Drupal 10

albeortev committed:
Issue #104: Drupal 10 require PHP >= 8.1

omarlopesino commented on composer.json:
```diff
@@ -33,14 +33,14 @@
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-composer-scaffold": "^10.0",
+        "drupal/core-recommended": "^10.0",
+        "drupal/drupal-extension": "^5.0@alpha",
```
- Drupal extension is  a dev library and it is being not required as require-dev
- Drupal extension is not required by required by the boilerplate but by nuvoleweb/drupal-behat which is already required by metadrop/behat-contexts, we should not require it as we rely on metadrop/behat-contexts to require the complete stack.

```suggestion
```

albeortev replied:
Thanks for the review, I remove the requirement of the package, it has been added unintentionally when testing the local functionality.

albeortev committed:
Issue #104: Remove required package directly in the template

Drupal extension is not required by required by the boilerplate but by
nuvoleweb/drupal-behat which is already required by
metadrop/behat-contexts, we should not require it as we rely on
metadrop/behat-contexts to require the complete stack.

albeortev committed:
Issue #104: Don't use Goutte with Drupal Extension v5.0.0alpha1

As of Drupal Extension version v5.0.0alpha1, Goutte is no longer used
and is replaced by Browserkit.
https://github.com/jhedstrom/drupalextension/commit/2ab66a7eae53ae4e1e5824edaa72e98039b084db

Eduardo-Morales-Alberti committed:
Require drupal dev drupal 10

Eduardo-Morales-Alberti committed:
drupal 10

Eduardo-Morales-Alberti committed:
drupal 10

albeortev committed:
Drupal 10

omarlopesino commented:
@albeorte96 any advances on this?

albeortev committed:
Issue #104: Set metadrop/drupal-dev version Drupal 10 compatibility

albeortev commented:
> @albeorte96 any advances on this?

Some related pending changes have been adjusted.
It is ready for testing

omarlopesino commented:
@albeorte96 then please mark it as a ready for review, thanks!

juanjol commented:
Tested and works great. Changes looks good to me, maybe this could be the start of Boilerplate 3.0?

juanjol commented:
Now that it is approved, should it be merged into a new branch or release? Maybe 3.0-rc, or beta?
